### PR TITLE
[Fixes #247] ConsoleLogger doesn't use LoggerFactory.MinimumLevel by …

### LINKED ISSUE #247: ConsoleLogger doesn't use LoggerFactory.MinimumLevel by default
See this line: https://github.com/aspnet/Logging/blob/dev/src/Microsoft.Framework.Logging.Console/ConsoleLoggerFactoryExtensions.cs#L16

This means the following doesn't result in verbose messages being emitted:

``` csharp
loggerFactory.MinimumLevel = LogLevel.Verbose;
loggerFactory.AddConsole();
```

### PR DESCRIPTION
**Pull request by kichalla**
@Eilon @AspNetSmurfLab Currently, the default level on logger factory is `Verbose`, which we want to change to `Information`, right? https://github.com/aspnet/Logging/blob/dev/src/Microsoft.Extensions.Logging/LoggerFactory.cs#L34

### CODE REVIEW

**Comment by Eilon** [comment]
Wait, why do we want to change it? What was wrong?

@DamianEdwards

File: src/Microsoft.Extensions.Logging.Console/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -3,6 +3,8 @@
 
 using System.Reflection;
 using System.Resources;
+using System.Runtime.CompilerServices;
 
 [assembly: AssemblyMetadata("Serviceable", "True")]
-[assembly: NeutralResourcesLanguage("en-us")]
\ No newline at end of file
+[assembly: NeutralResourcesLanguage("en-us")]
+[assembly: InternalsVisibleTo("Microsoft.Extensions.Logging.Test")]
```
Comment (Eilon) [inline src/Microsoft.Extensions.Logging.Console/Properties/AssemblyInfo.cs]: IVT is only from a product assembly to its own unit test assembly. Never use across misc assemblies.

**Comment by DamianEdwards** [comment]
Not sure about changing the default level. Just wanted to ensure the extension methods to add providers honored the default level.

**Comment by Eilon** [comment]
I'm thoroughly confused. What is "the default level"? The minimum level has nothing to do with any defaults - it just says that any logs below the minimum will not even be seen by loggers, no matter how hard they try.

What exactly was the bug or undesirable behavior?

**Comment by kichalla** [comment]
@Eilon Before my current change the default for the Console, Debug and EventLog was Information and now since we would default to logger factory's level, it would be Verbose...do we want Verbose to be the default?

File: src/Microsoft.Extensions.Logging.Console/Properties/AssemblyInfo.cs (hunk 1 above)
Comment (kichalla) [inline src/Microsoft.Extensions.Logging.Console/Properties/AssemblyInfo.cs]: Ok, but there is only test project for all these Console, Debug and EventLog projects...are you suggesting to create different test projects respectively?

**Comment by kichalla** [comment]
@Eilon fyi https://github.com/aspnet/Logging/issues/247

**Comment by Eilon** [comment]
Unfortunately, that bug is not a bug, it's by design.

**Comment by DamianEdwards** [comment]
The design is bad then. The code in the bug should result in logging at the default level of the factory IMO

**Comment by Eilon** [comment]
There's *no such thing* as a default level for the factory. The property is called `MinimumLevel`. Adding a `DefaultLevel` would be a separate feature request.

**Pull request closed without merging**